cpp/benchmarks/io/parquet/parquet_reader.cpp

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
Please note this file has been removed in https://github.com/rapidsai/cudf/pull/11611.